getting-started/setup-building.rst

```diff
@@ -419,7 +419,17 @@ For example, with **Homebrew**, install the dependencies::
 
     $ brew install pkg-config openssl@1.1 xz gdbm tcl-tk
 
-Then, for Python 3.10 and newer, run ``configure``::
+Then, for Python 3.11 and newer, run ``configure``::
+
+    $ GDBM_CFLAGS="-I$(brew --prefix gdbm)/include" \
+      GDBM_LIBS="-L$(brew --prefix gdbm)/lib -lgdbm" \
+      LIBLZMA_CFLAGS="-I$(brew --prefix xz)/include" \
+      LIBLZMA_LIBS="-I$(brew --prefix xz)/lib -llzma" \
+      PKG_CONFIG_PATH="$(brew --prefix tcl-tk)/lib/pkgconfig" \
+      ./configure --with-pydebug \
+                  --with-openssl="$(brew --prefix openssl@1.1)"
+
+For Python 3.10::
```

Comment:
I've finally made some time to test and experiment with Homebrew on both Intel and Apple Silicon Macs with 3.12 through 3.7. The good news is that I think we only need two, somewhat simpler recipes now:

for 3.11+:

```
GDBM_CFLAGS="-I$(brew --prefix gdbm)/include" \
GDBM_LIBS="-L$(brew --prefix gdbm)/lib -lgdbm" \
./configure --with-pydebug \
                  --with-openssl="$(brew --prefix openssl@1.1)"

```
For 3.10 through 3.7:

```
CPPFLAGS="-I$(brew --prefix gdbm)/include -I$(brew --prefix xz)/include" \
LDFLAGS="-L$(brew --prefix gdbm)/lib -L$(brew --prefix xz)/lib" \
./configure --with-pydebug \
                  --with-openssl="$(brew --prefix openssl@1.1)" \
                  --with-tcltk-libs="$(pkg-config --libs tcl tk)" \
                  --with-tcltk-includes="$(pkg-config --cflags tcl tk)"

```

(The `--with-tcltk*` options aren't needed for 3.10, just for 3.9 and earlier, but they don't hurt for 3.10.)

This was with the current version of Homebrew and macOS (Ventura 13.3.1).  Additional testing in other environments would be good :) In particular, I did not attempt to test using `brew link`. And, obviously, this is a basic example; there's a lot more we *could* say about overriding Apple-supplied packages, licenses, etc. But that's something for another day.

The MacPorts recipes should also be similarly updated.

`sudo port install pkgconfig openssl11 xz gdbm tcl tk +quartz
`

3.11+:

```
GDBM_CFLAGS="-I$(dirname $(dirname $(which port)))/include" \
GDBM_LIBS="-L$(dirname $(dirname $(which port)))/lib -lgdbm" \
./configure --with-pydebug \
                  --with-openssl="$(dirname $(dirname $(which port)))/libexec/openssl11"

```

3.10 through 3.7:

```
CPPFLAGS="-I$(dirname $(dirname $(which port)))/include" \
LDFLAGS="-L$(dirname $(dirname $(which port)))/lib" \
./configure --with-pydebug \
                  --with-openssl="$(dirname $(dirname $(which port)))/libexec/openssl11" \
                  --with-tcltk-libs="$(pkg-config --libs tcl tk)" \
                  --with-tcltk-includes="$(pkg-config --cflags tcl tk)"

```

I didn't want to screw up the discussion here by pushing an update to the PR but I could do that if it would help.

Reply:
Thanks, Ned! I'll have a look at this tonight. I can update the PR and the docs (I'll request your review).

Reply:
Hey @erlend-aasland , did you ever get a chance to get back to this? Thanks!

Reply:
No, I got bogged down with other stuff and it slipped away from my mind. Thanks for the reminder!

Reply:
I know that very well :sweat_smile:

Reply:
Tested on 3.7 through main on macOS 13.4; works swell!